CommunityToolkit.Common/Attributes/NotNullIfNotNullAttribute.cs

```diff
@@ -1,6 +1,6 @@
 // Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
+// The .NET Foundation licenses this file under the MIT license.
```

Comment:
Please undo these changes with a rebase. I said it was great to switch to .editorconfig, but I didn't say we should change the actual header. That's the same standard one across repos, please keep it that way. Even if we did want to change it, it shouldn't be done in the same PR.

Small note, since this keeps happening: contributions are welcome and I do appreciate your help, but please stop adding more changes to every PR that we never discussed. This just makes the review process unnecessarily long due to all the back and forth. Thank you 🙂

Reply:
Sure, Will move it to a new PR post discussion. I've marked it as Draft because of that. I pushed here just to validate it via CI.